docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,673 @@
+
+The data we will be using through out this tutorial is a study about how in a conversation the change of a speaker or the change of precedents (which are patterns of word usage to discribe an object, e.g. one can refer to the same object "white shoes", "runners", "sneakers") affects the understanding.
+
+Objects are presented on a screen while participants listen to instructions to move the objects around. Participants eye movements are tracked.
+The dependent variable is response time, defined as the latency between the onset of the test description and the moment at which the target was selected.
+The independent variables are speaker (old vs. new), precedents (maintain vs. break)  and cognitive load (a secondary memory task).
+
+We have to load the data and define some characteristics like the contrasts and the underlying model.
+
+Load existing data:
+```@example Main
+kb07 = MixedModels.dataset("kb07");
+```
+
+Set contrasts:
+```@example Main
+contrasts = Dict(:spkr => HelmertCoding(),
+                 :prec => HelmertCoding(),
+                 :load => HelmertCoding());
+```
+
+The chosen LMM for this dataset is defined by the following model formula:
+```@example Main
+kb07_f = @formula( rt_trunc ~ 1 + spkr+prec+load + (1|subj) + (1+prec|item) );
+```
+
+Fit the model
+```@example Main
+kb07_m = fit(MixedModel, kb07_f, kb07, contrasts=contrasts);
+print(kb07_m)
+```
+
+## **1.2 Simulate from existing data with same model parameters**
+
+We will first look at the power of the dataset with the same parameters as in the original data set. This means that each dataset will have the exact number of observations as the original data. Here, we use the model `kb07_m` we fitted above to our dataset `kb07`.
+
+You can use the `parameparametricbootstrap()` function to run `nsims` iterations of data sampled using the parameters from `kb07_m`.
+Set up a random seed to make the simulation reproducible. You can use your favourite number.
+
+To use multithreading, you need to set the number of cores you want to use.
+E.g. in Visual Studio Code, open the settings (gear icon in the lower left corner or cmd-,) and search for "thread".
+Set `julia.NumThreads` to the number of cores you want to use (at least 1 less than your total number).
+
+
+Set random seed for reproducibility:
+```@example Main
+rng = StableRNG(42);
+```
+
+Run nsims iterations:
+```@example Main
+kb07_sim = parametricbootstrap(rng, nsims, kb07_m, use_threads = false);
+```
+**Try**: Run the code above with or without `use_threads = true`.
+
+The output DataFrame `kb07_sim` contains the results of the bootstrapping procedure. 
+
+```@example Main
+df = DataFrame(kb07_sim.allpars);
+first(df, 9)
+nrow(df)
+```
+
+The dataframe df has 9000 rows: 9 parameters, in 1000 iterations.
+
+TODO: NEED HELP: is there a facet_wrap function in Gadfly? 
```

Comment:
There's no `facet_wrap`, but there is something like `facet_grid`:

http://gadflyjl.org/stable/lib/geometries/#Gadfly.Geom.subplot_grid

Reply:
Thanks for this advice. As it is not substantial for this tutorial maybe we keep it as four separate plots.